wsmaster/che-core-api-devfile/README.md

```diff
@@ -64,6 +64,32 @@ chectl workspace:start --devfile=devfile.yaml
 Please note that currently this way only works for the local (same machine) devfiles - URL can't be used here atm.
 
 
+### Project details
+A single devfile can specify several projects. For each project, one has to specify the type of the
+source repository, its location and optionally also the directory to which the project should be 
```

Comment:
I think it is not clear that this `clonePath` is a relative path to projects source. Is it relative?

Reply:
This is specified in the schema docs. I didn't want this to be a full reference, but rather a kind of introductory text.